### PR TITLE
Backfilling Airflow DAGs using Marquez

### PR DESCRIPTION
**Pull request by wslulciuc**
This PR adds the a draft version of the _Backfilling Airflow DAGs using Marquez_ blog.

### CODE REVIEW

**Review by collado-mike — COMMENTED** [review]
I'll need to get a link to your discussion of the lineage API response to put in my post- you went in depth exactly where I started glossing over details 😂

File: contents/blog/backfilling-airflow-dags-using-marquez/index.mdx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    | jq -r --arg NODE_ID "${node_id}" '.graph[] | select(.id==$NODE_ID) | .outEdges[].destination'))
+  for out_edge in "${out_edges[@]}"; do
+    # Run backfill if out edge is a job node (i.e. <dataset> => <job>)
+    if [[ "${out_edge}" = job:* ]]; then
+      dag_id="${out_edge##*:}"
+      echo "backfilling ${dag_id}..."
+      airflow backfill --start_date "${start_date}" --end_date "${start_date}" "${dag_id}"
+    fi
+    # Follow out edges downstream, recursively
+    backfill_downstream_of "${out_edge}"
```
[the remark this answers is not in the document]
Comment (wslulciuc) [reply contents/blog/backfilling-airflow-dags-using-marquez/index.mdx:179]: Thanks for the suggestion, @collado-mike! I'll try and make the suggested change before the blog is posted!

**Comment by rossturk** [comment]
I have added some images - take a look @wslulciuc and let me know what you think.

**Review by rossturk — APPROVED** [review]
This biog is great!

**Comment by wslulciuc** [comment]
> I have added some images - take a look @wslulciuc and let me know what you think.

Images look great. Thanks, @rossturk!

**Comment by wslulciuc** [comment]
> I'll need to get a link to your discussion of the lineage API response to put in my post- you went in depth exactly where I started glossing over details 😂

Yes! You'd almost think it was planned!

**Pull request merged**